.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -8,11 +6,17 @@
 - [ ] **Submitter**: Include the JIRA issue number in the PR description
 - [ ] **Submitter**: Add description or comments on the PR explaining the hows/whys (if not obvious)
 - [ ] **Submitter**: If PR includes new or changed db queries, include the explain plans in the description
+- [ ] **Submitter**: Update FISMA documentation if changes to:
+  * Authentication
+  * Authorization
+  * Encryption
+  * Audit trails
+- [ ] **Submitter**: If you're adding new libraries, sign us up to security updates for them
 - [ ] Tell ![](http://i.imgur.com/9dLzbPd.png) that the PR exists if he wants to look at it **(apply requires_doge label)**
 - [ ] Anoint a lead reviewer (LR). **Assign PR to LR**
 - [ ] **LR**: Initial review by LR and others.
-- [ ] Comment / review / update cycle:
-  * Rest of team may comments on PR at will
+- [ ] Review / update cycle:
```

Comment:
While you're here, I always stumble on this one because it looks like a reminder, not a task. I don't ever really know what it means to check this off.